packages/components/psammead-sitewide-links/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-sitewide-links/README.md",
   "dependencies": {
-    "@bbc/gel-foundations": "^0.3.0",
+    "@bbc/gel-foundations": "^3.0.0",
```

Comment:
The `CHANGELOG` for the `psammead-sitewide-links` package will need to be also updated.